src/pe_reports/pages.py

```diff
@@ -65,9 +66,16 @@ def buildAppendixList(df):
     return html
 
 
+def sanitize_uid(string):
+    """Remove special characters from uids."""
+    return re.sub(r"[^a-zA-Z0-9\-]", "", string)
+
+
```

Comment:
Can you reuse `db_query.sanitize_uid()` here as well?

Reply:
Yes missed that, commit b04a2ff821024367373374139f3871c56a496b99

Reply:
Yes! missed it, commit b04a2ff821024367373374139f3871c56a496b99, then fixed some build and linting issues